src/test/bridges/yearn/YearnBridgeE2E.t.sol

```diff
@@ -154,8 +178,20 @@ contract YearnBridgeE2ETest is BridgeTestBase {
         assertEq(inputAssetAMid, inputAssetABefore - _depositAmount, "Balance missmatch after deposit");
         assertGt(outputAssetAMid, outputAssetABefore, "No change in output asset balance after deposit");
 
-        _withdrawAmount = outputAssetAMid;
-        bridgeCallData = encodeBridgeCallData(id, depositOutputAssetA, emptyAsset, depositInputAssetA, emptyAsset, 1);
+        // Move some funds to strategies to have underlying balance be less than the required.
```

Comment:
I don't fully understand this. Is this supposed to cause revert or something else? Please expand a bit on what this will achieve.